bee-message/src/output/feature_block/timelock_milestone_index.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{milestone::MilestoneIndex, Error};
+
+use bee_byte_cost::ByteCost;
+use bee_common::packable::{Packable, Read, Write};
+
+use derive_more::From;
+
+/// Defines a milestone index until which the output can not be unlocked.
+#[derive(ByteCost, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, From)]
+#[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
+pub struct TimelockMilestoneIndexFeatureBlock(
```

Comment:
This was merged with TimeLockUnixFeatureBlock to a single 
```Rust
pub struct TimelockUnlockCondition {
    // The milestone index starting from which the output can be consumed.
    milestone_index: MilestoneIndex,
    // Unix time, seconds since unix epoch, starting from which the output can be consumed.
    timestamp: u32,
}
```